literato-backend/config/config.json

```diff
@@ -9,7 +9,7 @@
   "test": {
     "username": null,
     "password": null,
-    "database": "literato_test",
+    "database": "literato_development",
```

Comment:
you probably do want a seperate test repo - otherwise running the tests will mess up your dev database